integration/tests/cook/test_basic.py

```diff
@@ -33,6 +34,9 @@ def test_basic_submit(self):
         job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
         self.assertEqual('success', job['instances'][0]['status'])
         self.assertEqual(False, job['disable_mea_culpa_retries'])
+        if util.is_using_cook_executor(self.cook_url):
+            self.assertEqual(0, job['instances'][0]['exit_code'])
+            self.assertTrue(bool(job['instances'][0]['sandbox_directory']))
```

Comment:
@shamsimam did you push the update, it looks like this comment is not addressed.